bin/_help

```diff
@@ -7,34 +7,33 @@
 source "/usr/local/bin/_common"
 
 print_common_usage() {
-  echo "Usage: nib ${1} [SERVICE]"
+  echo "Usage: nib $COMMAND [SERVICE]"
 }
 
 print_help_options() {
-  echo "Options:
-  -h, --help      Print usage
-      --version   Print version information
-"
```

Comment:
This prior implementation was only useful for `nib --help`. This has been moved there making this function more flexible.